tools/interface/ontology.py

```diff
@@ -46,16 +49,50 @@ def GetEntitiesFromFields(self, field_list: List[StandardField], general_type: s
         """
         pass
 
-    def IsTypeValid(self, entity_type: EntityType) -> bool:
-        """Validates the existence of a type in the ontology"""
-        pass
-
     def IsFieldValid(self, field: StandardField) -> bool:
         """A method to validate a field name against the ontology"""
         pass
 
-    def IsSubfieldValid(self, subfield: StandardSubfield) -> bool:
-        """A method to validate a subfield name against the ontology"""
-        pass
 
+class StandardField(object):
+    """
+    A class to represent a generic field without increment or optionality
+
+    Args:
+        namespace_name: the name of the field's namespace as a string
+        field_name: the name of the field as a string
+
+    Attributes:
+        namespace: the name of the namespace as a string
+        name: the field name as a string
+
+    returns:
+        An instance of the StandardField class
 
+    """
+
+    def __init__(self, namespace_name: str, field_name: str):
+        super().__init__()
+        self.namespace = namespace_name
+        self.name = field_name
+
+class EntityTypeField(StandardField):
+    """
+    A class to represent a field assigned to a type and extends StandardField
+
+    Args:
+        increment: the increment of the field as a string
+        is_required: optionality of the field as a boolean
+
+    Attributes:
+        increment: the increment of the field under a type
+        is_required: optionality of the field relative to it's Entity Type# Configure the Azure provider
+
+    Returns:
+        An instance of the EntityTypeField class
+    """
+
+    def __init__(self, increment: str = '', is_required: bool):
```

Comment:
the init needs arguments for the namespace and standard field name, then you want to call super with these arguments.